content/docs/pulumi-cloud/oidc/provider/aws.md

```diff
@@ -99,7 +99,7 @@ The OIDC subject claim for this environment would be `pulumi:environments:pulumi
 ```json
 "Condition": {
   "StringEquals": {
-    "api.pulumi.com/oidc:aud": "contoso",
+    "api.pulumi.com/oidc:aud": "aws:contoso",
```

Comment:
Will we need this for azure/gcp too?

Reply:
No those docs don't have codeblocks that reference the audience